hugegraph-core/src/main/java/org/apache/hugegraph/util/StringEncoding.java

```diff
@@ -137,18 +129,23 @@ public static byte[] compress(String value) {
     }
 
     public static byte[] compress(String value, float bufferRatio) {
-        BytesBuffer buf = LZ4Util.compress(encode(value), BLOCK_SIZE,
-                                           bufferRatio);
-        return buf.bytes();
+        try (BytesBuffer buf = LZ4Util.compress(encode(value), BLOCK_SIZE, bufferRatio)) {
```

Comment:
keep the origin logic, or split into a new commit?

Reply:
this commit will remove all TODOs after fix them & contains current commit

but we could not merge it now